go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
+	github.com/zalando/go-keyring v0.1.1
```

Comment:
I remember having this discussion with @EngHabu  where we didn't want to add this dependency in stdlib and keep it only in flytectl where we used the keyring 
Now that we have flyteadmin also needing this dependency, is it worth moving this to stdlib and also Token implemetation

Reply:
no longer relevant - marking resolved